pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/ingress_reconciler.go

```diff
@@ -379,12 +405,14 @@ func (ir *IngressReconciler) Reconcile(isvc *v1beta1.InferenceService) error {
 				path = constants.PredictPath(isvc.Name, constants.ProtocolV2)
 			}
 		}
-		isvc.Status.Address = &duckv1.Addressable{
-			URL: &apis.URL{
-				Host:   network.GetServiceHostname(isvc.Name, isvc.Namespace),
-				Scheme: "http",
-				Path:   path,
-			},
+		if deploymentMode != constants.ModelMeshDeployment || isvc.Spec.Transformer != nil {
```

Comment:
Why this is specific to ModelMesh?

Reply:
It is because the field is updated by ModelMesh already. We don't want to overwrite it here.